packages/next-on-pages/src/buildApplication/processVercelFunctions/invalidFunctions.ts

```diff
@@ -46,7 +46,10 @@ export async function checkInvalidFunctions(
 	await tryToFixInvalidFuncsWithValidIndexAlternative(collectedFunctions);
 	await tryToFixInvalidDynamicISRFuncs(collectedFunctions);
 
-	if (collectedFunctions.invalidFunctions.size > 0) {
+	if (
+		collectedFunctions.invalidFunctions.size > 0 &&
+		!opts.ignoreInvalidFunctions
```

Comment:
my concern with this approach is that people may find and use `--force` because they get this error, without actually accounting for the routes properly... In your case, you want to host these routes elsewhere, but I imagine there'll be people who don't realise they need to actually specify these routes should execute somewhere else, and expect it to just work by _forcing_ a build.

Reply:
Same. I don't enjoy the thought of forcing a build.

I worry that the alternative of trying to better evaluate the next.config to determine if the function is actually invalid may be fraught with difficulty (I've not looked too closely, so I'll defer to your judgement). Is this what you had in mind?

Would printing a warning instead of exiting be sufficient? So the errors aren't swept under the rug.

From a connotation perspective, I find `force` to be ok, but I'm happy to rename the argument as well